Github action failing to clone a sub-tree with svn

**Impact of the bug**
WMCore CD pipeline

**Describe the bug**
Creating this bug because it is the second time this issue happens this week. You can see in this recent log:
https://github.com/dmwm/WMCore/actions/runs/6318198746/job/17156697219

where we fail to fetch a sub-tree from CMSKubernetes repository with error:
```
Building service: reqmgr2ms-pileup, with tag: 2.2.4rc7
svn: E170013: Unable to connect to a repository at URL 'https://github.com/dmwm/CMSKubernetes/trunk/docker/pypi/reqmgr2ms-pileup'
svn: E175002: Unexpected server error 500 'Internal Server Error' on '/dmwm/CMSKubernetes/trunk/docker/pypi/reqmgr2ms-pileup'
Error: Process completed with exit code 1.
```

The service that it fails is random, and upon one failure, the whole CD pipeline gets aborted.

Stack overflow does not suggest any real fix, but to retry:
https://stackoverflow.com/questions/72754026/error-500-checking-out-from-github-with-svn

**How to reproduce it**
Not sure

**Expected behavior**
We can try adding a retry logic, if it does not work, then I would suggest to stop relying on svn and fetch the necessary information in a different way (either with a `git clone` of the whole repo; or with curl).

**Additional context and error message**
None

--- a/.github/workflows/docker_images_template.yaml
+++ b/.github/workflows/docker_images_template.yaml
@@ -38,7 +38,14 @@ jobs:
           CERN_REGISTRY: registry.cern.ch
         run: |
           echo "Building service: ${SERVICE_NAME}, with tag: ${PYPI_TAG}"
-          svn checkout https://github.com/dmwm/CMSKubernetes/trunk/docker/pypi/${SERVICE_NAME}
+          for i in 1 2 3 4 5; do
+            svn checkout https://github.com/dmwm/CMSKubernetes/trunk/docker/pypi/${SERVICE_NAME}
+            if [ $? -eq 0 ]; then
+              break
+            fi
+            echo "Retry $i: SVN failed to fetch files. Sleeping 60 seconds..."
+            sleep 60
+          done
           cd ${SERVICE_NAME}
           echo "Retrieved Dockerfile with content:"
           cat Dockerfile

